### DISCUSSION TITLE
Cast storage not working as expected: Getting different output(null) while expecting the bytes output of myPassword

### DISCUSSION THREAD

**Discussion by skpkss** [foundry-rs/foundry#8739] [Help]
![q](https://github.com/user-attachments/assets/9cf18534-1c56-40f0-b00b-da9ee334b9ea)
 I am building on proof of code to show that private variable are not actually private. I want to display the byte representation of "myPassword", but when i am running the cast storage command it is displaying me the bytes of null instead of 'myPassword'. Can someone help me with this issue?

**Comment by urosognjenovic** [comment]
Can we see the ```PasswordStore``` contract?
